pom.xml

```diff
@@ -452,7 +452,7 @@
     <version.jacorb>2.3.1jboss.patch01-brew</version.jacorb>
     <version.org.jboss.openjdk-orb>8.0.8.Final</version.org.jboss.openjdk-orb>
     <version.javax.annotation>1.2</version.javax.annotation>
-    <version.javax.enterprise>1.1</version.javax.enterprise>
+    <version.javax.enterprise>1.2</version.javax.enterprise>
```

Comment:
Please can you explain why we needed to update this dependency?

Reply:
The javax.enterprise.cdi-api 1.1depends on the org.jboss.spec.javax.interceptor.jboss-interceptors-api_1.1_spec 1.0.0.Beta1 which is out of date and causes the ClassNotFound exception when building the javadoc of ArjunaJTA/cdi